rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -263,6 +263,13 @@ def _create_networks(self, network_create_args):
     def _bind_ports(self, lports, sandboxes, port_bind_args):
         port_bind_args = port_bind_args or {}
         wait_up = port_bind_args.get("wait_up", False)
+        # "wait_sync" takes effect only if wait_up is True.
```

Comment:
@hzhou8 sorry for missing the condition check code below. If so, shall we move the above comment near ``if wait_up``?